DNN Platform/Admin Modules/Dnn.Modules.ConfigurationManager/XmlMerge.ascx

```diff
@@ -58,9 +58,9 @@
 	jQuery(function($) {
 		$('#<%=cmdSave.ClientID%>, #<%= cmdExecute.ClientID %>').dnnConfirm({
 			text: '<%= ConfirmText %>',
-			yesText: '<%= Localization.GetSafeJSString("Yes.Text", Localization.SharedResourceFile) %>',
-			noText: '<%= Localization.GetSafeJSString("No.Text", Localization.SharedResourceFile) %>',
-			title: '<%= Localization.GetSafeJSString("Confirm.Text", Localization.SharedResourceFile) %>'
+			yesText: '<%= HttpUtility.JavaScriptStringEncode("Yes.Text", Localization.SharedResourceFile) %>',
+			noText: '<%= HttpUtility.JavaScriptStringEncode("No.Text", Localization.SharedResourceFile) %>',
+			title: '<%= HttpUtility.JavaScriptStringEncode("Confirm.Text", Localization.SharedResourceFile) %>'
```

Comment:
When a resource file path is passed as the second parameter, `Localization.GetString` is called on the values before they're "encoded" (see https://github.com/dnnsoftware/Dnn.Platform/blob/f50786674d8e9e2566189f4cdf80e24c6f840e02/DNN%20Platform/Library/Services/Localization/Localization.cs#L1263)